app/main/db/queries.py

```diff
@@ -81,15 +83,53 @@ def build_fuzzy_search_query(query_string: str, sorting_orders=None):
         ).label("opening_date"),
     )
 
+    if transferring_body_id:
```

Comment:
not a big deal but was there a particular reason for moving this filter that depends on the user being a AAU or standard user from the route itself?

Reply:
there are two separate queries for search transferring body and search results summary, due to that added transferring_body_id as a filter which will be needed when accessing search transferring body screen